pinot-tools/src/main/java/org/apache/pinot/tools/service/PinotServiceManager.java

```diff
@@ -107,6 +108,7 @@ public String startRole(ServiceRole role, Map<String, Object> properties)
   public String startController(String controllerStarterClassName, PinotConfiguration controllerConf)
       throws Exception {
     LOGGER.info("Trying to start Pinot Controller...");
+    LOGGER.info(new PinotAppConfigs(controllerConf).getJvmInputArguments());
```

Comment:
I don't think we need to log here because it will be logged within the component starter